db/deploy/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Use the official lightweight Python image.
+FROM python:3.11-slim
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends gcc git ssh default-jdk wget unzip
+
+# Download and install Liquibase
+ENV LIQUIBASE_VERSION=4.26.0
+RUN wget https://github.com/liquibase/liquibase/releases/download/v${LIQUIBASE_VERSION}/liquibase-${LIQUIBASE_VERSION}.zip \
+    && unzip liquibase-${LIQUIBASE_VERSION}.zip -d /opt/liquibase \
+    && chmod +x /opt/liquibase/liquibase \
+    # Clean up to reduce layer size
+    && rm liquibase-${LIQUIBASE_VERSION}.zip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download the MariaDB JDBC driver
+ENV MARIADB_JDBC_VERSION=3.0.3
```

Comment:
Can you bubble up all the versions to the top so they're easy to scan?

Reply:
Done!